pkg/cloud/aws/services/ec2/ami.go

```diff
@@ -35,6 +35,10 @@ func amiName(platform, platformVersion, kubernetesVersion string) string {
 func (s *Service) defaultAMILookup(platform, platformVersion, kubernetesVersion string) (string, error) {
 	describeImageInput := &ec2.DescribeImagesInput{
 		Filters: []*ec2.Filter{
+			{
+				Name:   aws.String("owner-id"),
+				Values: []*string{aws.String("258751437250")},
```

Comment:
Can we constantize this with a comment to say whose account this is?

Reply:
i can include this commit in the following cleanup PR instead since I need to clean this whole file up anyway